pkg/utils/experimentHelpers.go

```diff
@@ -7,14 +7,25 @@ import (
 )
 
 //SetValueFromChaosExperiment sets value in experimentDetails struct from chaosExperiment
-func (expDetails *ExperimentDetails) SetValueFromChaosExperiment(clients ClientSets) {
-	expDetails.SetImage(clients)
-	expDetails.SetArgs(clients)
-	expDetails.SetLabels(clients)
+func (expDetails *ExperimentDetails) SetValueFromChaosExperiment(clients ClientSets, engine *EngineDetails) error {
+	if err := expDetails.SetImage(clients); err != nil {
+		return err
+	}
+	if err := expDetails.SetArgs(clients); err != nil {
+		return err
+	}
+	// Get engineUID from the chaos-runner's label
+	if err := SetEngineUID(engine, clients); err != nil {
```

Comment:
Why are we adding this function here, can we take this in `SetValueFromChaosEngine`